package.json

```diff
@@ -23,7 +23,9 @@
   "dependencies": {
     "@nypl/dgx-svg-icons": "^0.3.9",
     "react": "^15.4.0",
-    "react-dom": "^15.4.0"
+    "react-dom": "^15.4.0",
+    "react-router": "^5.0.0",
```

Comment:
These don't belong here for reusable components and they're not being referenced in the app anyway.

Reply:
Turns out we do need `react-dom`; otherwise, this happens when you run the tests:
<img width="779" alt="Screen Shot 2019-04-17 at 5 47 54 PM" src="https://user-images.githubusercontent.com/42178216/56323311-6d856680-6139-11e9-8121-2506133a4f2e.png">

Reply:
I meant for `react-router`.